hacks/update-jenkins-plugins/collect-jenkins-plugins.sh

```diff
@@ -100,11 +100,8 @@ get_plugin() {
     plugin=$(echo "${plugin_line}:" | cut -d : -f 1)
     plugin_version=$(echo "${plugin_line}:" | cut -d : -f 2)
 
-    if [ -z "${plugin_version}" -o "${plugin_version}" == "latest" ]; then
-        plugin_url="https://updates.jenkins-ci.org/latest/${plugin}.hpi"
-    else
-        plugin_url="https://updates.jenkins-ci.org/download/plugins/${plugin}/${plugin_version}/${plugin}.hpi"
-    fi
+    plugin_url="https://ftp.belnet.be/pub/jenkins/plugins/${plugin}/${plugin_version}/${plugin}.hpi"
```

Comment:
Is there a reason why we are switching to a mirror, vs. using the official `jenkins-ci.org` repo?

Reply:
yes; so that we can open a single hole in our firewall, instead of an ever-changing list of holes.